### DISCUSSION TITLE
Palestra - Vitor Mendrone sobre Acessibilidade

### DISCUSSION THREAD

**Discussion by fabiootc** [frontendbr/forum#127] [Discussão]
Palestra do Vitor Mendrone sobre acessibilidade, está rolando agora, bem interessante, assite lá:
http://www.fastcommerce.com.br/sistema/ack.asp?IDLoja=1&IDLogEM=77822&IDAL=8510190&origem=emailmkt&URL=https%3A%2F%2Feventos%2Elojavirtualfc%2Ecom%2Ebr%2Fpalestra

**Comment by felipefialho** [comment]
@Mendrone :stuck_out_tongue:

**Comment by vitor-mendrone-deel** [comment]
Aee @fabiootc valeu :D 

Foi uma palestra menos técnica, pra lojistas mesmo mas foi divertida :grin: 

Não precisei nem pagar pela propaganda :stuck_out_tongue:

**Comment by fabiootc** [comment]
Pois é mano! Mas achei bem interessante, tenho estudado bastante acessibilidade recentemente até porque tenho desenvolvido bastante lojas de Farmácia.

Muito desenv. ignora isso, e são coisas simples de serem aplicadas no dia a dia de qualquer projeto. Sem contar o puta valor que agrega para loja/desev/user.
